finalize/finalizeStep.py

```diff
@@ -5,9 +5,13 @@
 class finalizeStep():
     # class constructor
 
-    def __init__(self, id, manifestMetadata):
+    def __init__(self, id, event):
```

Comment:
down below you are sending the event-config data from the hash not the event itself.  I think that this should either be eventConfig or the bottom should be changed to pass the event for consistency